pac/creating-pipeline-runs-pac.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="creating-pipeline-runs-pac"]
+= Creating pipeline runs in {pac}
+:context: creating-pipeline-runs-pac
+
+toc::[]
+
+[role="_abstract"]
+After integrating {pac} with your repository provider and defining your particular repository using the `Repository` custom resource (CR), you can create pipeline run definitions in your repository.
+
+include::modules/op-creating-pipeline-run-pac.adoc[leveloffset=+1]
+
+include::modules/op-parameters-pipeline-run-using-pipelines-as-code.adoc[leveloffset=+1]
+[role="_additional-resources"]
+[id="additional-resources-parameters-pipelinerun_{context}"]
+.Additional resources
+* xref:../pac/using-pipelines-as-code-repos.adoc#scoping-github-token_using-pipelines-as-code-repos[Scoping the GitHub token to additional repositories]
+
+
+include::modules/op-pipelines-as-code-resolver-annotations.adoc[leveloffset=+1]
+
+include::modules/op-using-remote-task-annotations-with-pipelines-as-code.adoc[leveloffset=+2]
+
+include::modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc[leveloffset=+2]
+
+include::modules/op-matching-pipeline-run-using-pipelines-as-code.adoc[leveloffset=+1]
+
+include::modules/op-filtering-pipeline-run-using-pipelines-as-code.adoc[leveloffset=+1]
+
+
+[role="_additional-resources"]
+[id="additional-resources-filtering-pipelinerun_{context}"]
+.Additional resources
+
+* link:https://github.com/google/cel-spec/blob/master/doc/langdef.md[CEL language specification]
```

Comment:
I don't want to block a merge over this, but did you get PX approval for this github link in the past? I see that this content is moved from another assembly. PTAL at the peer review checklist and the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites). If did not get approval in the past, please get approval or delete the link in a follow up PR.